docs/getting-started/osx.rst

```diff
@@ -1,8 +1,230 @@
+ - Use brew to install Mono
+
+    .. code-block:: console
+
+        $ brew install mono
+
+ - Run the dnvm
+
+    .. code-block:: console
+
+        $ curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | bash && source ~/.dnx/dnvm/dnvm.sh
+
+ - Verify dnvm has the latest version
+    
+    .. code-block:: console
+
+        $ dnvm upgrade
+
+If you have trouble installing dnvm, consult this `Getting Started guide <http://dotnet.github.io/core/getting-started/>`_.
+
+
+Create a new project
+--------------------
+ - Create a new folder for your project. All files for the project should be contained in this folder. 
+ - Open a new Terminal window and navigate to your project folder. All commands listed below are executed from the project root folder.
+ - Create a new file ``project.json`` with the following contents
+
+    .. literalinclude:: x-plat/sample/src/ConsoleApp/project.json
+            :language: json
+            :linenos:
+ 
+ - Execute the following command to install the packages required for this project, including EntityFramework 7 and all its dependencies.
+
+    .. code-block:: console
+
+        $ dnu restore
+
+ - Create a new file named ``Program.cs``. Add the following contents to
+
+    .. code-block:: c#
+        :linenos:
+
+        // Program.cs
+        using System;
+
+        namespace ConsoleApp
+        {
+            public class Program
+            {
+                public void Main(string[] args)
+                {
+                    Console.WriteLine("Hello dnx!");
+                }
+            }
+        }
+
+To verify that this project has all dependencies and packages installed, perform the following steps:
+
+ - Verify execution of the program works by running ``dnx . run``.
+
+    .. code-block:: console
+
+        $ dnx . run
+        Hello dnx!
+ - Verify that Entity Framework is installed by running ``dnx . ef``.
+
+    .. code-block:: console
+
+        $ dnx . ef
+
+                             _/\__
+                       ---==/    \\
+                 ___  ___   |.    \|\
+                | __|| __|  |  )   \\\
+                | _| | _|   \_/ |  //|\\
+                |___||_|       /   \\\/\\
+
+        Usage: ef [options] [command]
+
+        Options:
+          -v|--version  Show version information
+          -h|--help     Show help information
+
+        Commands:
+          context    Commands to manage your DbContext
+          migration  Commands to manage your Code First Migrations
+          revEng     Command to reverse engineer code from a database
+          help       Show help information
+
+        Use "ef [command] --help" for more information about a command.
+
+
+Begin using EF
+--------------
+With this new project, you are ready to begin using Entity Framework.
+We will create a simple console application that allows us to write a 
+blog post from the command line.
+
+ - Create a new file called ``BloggingModel.cs``
```

Comment:
Call this Model.cs and just list the entire code in one block (to align with other walkthroughs).
http://ef.readthedocs.org/en/latest/getting-started/full-dotnet.html#create-your-model